lib/signin.ts

```diff
@@ -9,11 +9,14 @@ export const setToken = async (token: string) => {
 };
 
 export default async function signin(email: string, password: string) {
-  let token = await backendFetch<SignInDTO>('POST', 'account/login', {
-    email: email,
-    password: password,
-  });
-
-  await setToken(token);
-  return token;
+  try {
+    let token = await backendFetch<SignInDTO>('POST', 'account/login', {
+      email: email,
+      password: password,
+    });
+    await setToken(token);
+    return true;
+  } catch (e) {
+    return false;
+  }
 }
```

Comment:
Should probably be moved up a level, letting the function itself handle error handling is kinda weird